modules/ROOT/pages/migration/index.adoc

```diff
@@ -20,6 +20,42 @@ npm update @neo4j/graphql
 
 Here is a list of all the breaking changes from version 5.0.0 to 6.0.0.
 
+=== The deprecated `_NOT` filters are no longer supported
+
+The deprecated `_NOT` filters such as:
```

Comment:
is the list below complete? if so, remove "such as". i'd also rather not split the sentence with the list.

maybe:

```suggestion
The following deprecated `NOT` filters are removed from the schema since they are no longer supported:
```